receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -40,6 +56,34 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: [topic]
+  kafka.topic.replication_factor:
+    enabled: true
+    description: replication factor of a topic.
+    unit: "1"
+    gauge:
+      value_type: int
+    attributes: [topic]
+  kafka.topic.log_retention_ms:
+    description: log retention period of a topic (ms), Default value is 7 days (604800000ms).
+    unit: ms
```

Comment:
I believe from [semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units) this unit should be in seconds.

Reply:
Thanks, I have changed the units to follow standard norms, Also made appropriate time conversions from logic side.